docs/architecture/modernize-desktop/deploy-modern-applications.md

```diff
@@ -0,0 +1,220 @@
+
+### Tools
+
+#### Windows Application Packaging Project
+
+You can use the **Windows Application Packaging Project** project in Visual Studio to generate a package for your desktop app. Then, you can publish that package to the Microsoft Store or sideload it onto one or more PCs.
+
+#### MSIX Packaging Tool
+
+The MSIX Packaging Tool enables you to repackage your existing Win32 applications to the MSIX format. It offers both an interactive UI and a command line for conversions and gives you the ability to convert an application without having the source code.
+
+![MSIX Packaging Tool](media\deploying-modern-desktop-applications\msix-packaging-tool.png)
+
+#### Package Support Framework
+
+The Package Support Framework is an open-source kit that helps you apply fixes to your existing Win32 application when you don't have access to the source code, so that it can run in an MSIX container. The Package Support Framework helps your application follow the best practices of the modern runtime environment.
+
+#### App Installer
+
+App Installer allows Windows 10 apps to be installed by double-clicking the app package. This means that users don't need to use PowerShell or other developer tools to deploy Windows 10 apps. The App Installer can also install an app from the web, optional packages, and related sets.
+
+## How to create an MSIX package from an existing Win32 desktop application
+
+Let's go through the process to create an MSIX package from an existing Win32 application. In this example, we'll use a Windows Forms app.
+
+To start, add a new project to your solution, select the Windows Application Packaging Project and give it a name.
+
+![Adding new Windows Application Packaging Project](media\deploying-modern-desktop-applications\adding-packaging-project.png)
+
+You'll see the structure of the packaging project and note a special folder called Applications. Inside this folder you can specify which applications you want to include in the package, it can be more than one.
+
+![The structure of the packaging project](media\deploying-modern-desktop-applications\packaging-project.png)
+
+Right-click on the *Applications* folder and select the Windows Forms project you want to package from the Visual Studio solution.
+
+![Adding our Windows Forms project to Packaging Project](media\deploying-modern-desktop-applications\adding-our-project.png)
+
+At this point, you can compile and generate the package but let's take a look at a couple of things. To have a better user experience, Visual Studio can autogenerate all the visual assets a modern application needs to handle icons and tile assets for the tile bar and start menu. Opening the *Package.appxmanifest* file you can access the Manifest Designer and generate all the visual assets from a given image present on your project just by clicking *Create*.
+
+![Manifest Designer](media\deploying-modern-desktop-applications\manifest-designer.png)
+
```

Comment:
The previous image seems very busy. Not sure if we could retake it on a different scale where less scrollbars would be shown?